google/cloud/spanner_v1/_pandas_helpers.py

```diff
@@ -0,0 +1,76 @@
+import logging
+import six
+
+class pandas_df():
+    
+    def pd_dataframe(self):
```

Comment:
`self` doesn't really make much sense to me here. What type is `self`? Clearly it's not a `pandas_df` object.

Reply:
changed